recipes/net-snmp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+add_executable(test_package test_package.c)
+target_link_libraries(test_package PRIVATE CONAN_PKG::net-snmp)
```

`CONAN_PKG::` targets will be removed in the next future 

```suggestion
conan_basic_setup(TARGETS)
find_package(net-snmp REQUIRED)

add_executable(test_package test_package.c)
target_link_libraries(test_package PRIVATE net-snmp::net-snmp)
```